build.gradle

```diff
@@ -22,3 +26,8 @@ wrapper {
   gradleVersion = '3.2.1'
   distributionType = Wrapper.DistributionType.ALL
 }
+
+coveralls {
```

Comment:
I met build error when I put this to `findbugs/build.gradle` and couldn't understand its meaning. I will retry later.

One more: I forgot checking coverage in eclipse plugin project. I'll try to check coverage and [merge the result from multiple projects](https://github.com/kt3k/coveralls-gradle-plugin#use-with-mutli-project-build).